adapter/internal/discovery/xds/server_utils.go

```diff
@@ -33,7 +34,7 @@ func getEnvironmentsToBeDeleted(existingEnvs, deleteEnvs []string) (toBeDel []st
 	}
 	// otherwise delete env if it wished to
 	for _, existingEnv := range existingEnvs {
-		if arrayContains(deleteEnvs, existingEnv) {
+		if stringutils.ArrayContains(deleteEnvs, existingEnv) {
```

Comment:
existing `stringutils.StringInSlice` and new utils`stringutils.ArrayContains` are same. shall we use the existing one?